hazelcast/src/main/java/com/hazelcast/config/WanTargetClusterConfig.java

```diff
@@ -24,6 +24,16 @@
 public class WanTargetClusterConfig {
 
     private static final WanAcknowledgeType DEFAULT_ACK_TYPE = WanAcknowledgeType.ACK_ON_OPERATION_COMPLETE;
+    private static final WanQueueFullBehavior QUEUE_FULL_BEHAVIOR = WanQueueFullBehavior.DISCARD_AFTER_MUTATION;
+    private static final int DEFAULT_BATCH_SIZE = 500;
+    private static final long DEFAULT_BATCH_MAX_DELAY_MILLIS = 1000;
+    private static final int DEFAULT_QUEUE_CAPACITY = 10000;
+    private static final long DEFAULT_RESPONSE_TIMEOUT_MILLIS = 60000;
```

Comment:
I see those final statics not used anywhere else so what is the point to declare them here and then assing to class fields?